module/documents/item.mjs

```diff
@@ -2061,6 +2064,21 @@ export default class Item5e extends Item {
     return updates;
   }
 
+  /**
+   * Pre-creation logic for the automatic configuration of owned feature type Items.
+   * @param {object} data       Data for the newly created item.
+   * @param {boolean} isNPC     Is this actor an NPC?
+   * @returns {object}          Updates to apply to the item data.
+   * @private
```

Comment:
These `_onCreate` methods should probably all be `@protected` but it's fine to leave this for now to be consistent with the others.